cmd/go-filecoin/address_integration_test.go

```diff
@@ -58,100 +55,48 @@ func TestWalletBalance(t *testing.T) {
 	require.NoError(t, err)
 
 	t.Log("[success] not found, zero")
-	balance := cmdClient.RunSuccess(ctx, "wallet", "balance", addr.String())
-	assert.Equal(t, "0", balance.ReadStdoutTrimNewlines())
+	var balance abi.TokenAmount
+	cmdClient.RunMarshaledJSON(ctx, &balance, "wallet", "balance", addr.String())
+	assert.Equal(t, "0", balance.String())
 
-	t.Log("[success] balance 9999900000")
-	balance = cmdClient.RunSuccess(ctx, "wallet", "balance", builtin.RewardActorAddr.String())
-	assert.Equal(t, "949999900000", balance.ReadStdoutTrimNewlines())
+	t.Log("[success] balance 1394000000000000000000000000")
+	cmdClient.RunMarshaledJSON(ctx, &balance, "wallet", "balance", builtin.RewardActorAddr.String())
+	assert.Equal(t, "1394000000000000000000000000", balance.String())
 
 	t.Log("[success] newly generated one")
-	addrNew := cmdClient.RunSuccess(ctx, "address", "new")
-	balance = cmdClient.RunSuccess(ctx, "wallet", "balance", addrNew.ReadStdoutTrimNewlines())
-	assert.Equal(t, "0", balance.ReadStdoutTrimNewlines())
+	var addrNew commands.AddressResult
+	cmdClient.RunMarshaledJSON(ctx, &addrNew, "address", "new")
+	cmdClient.RunMarshaledJSON(ctx, &balance, "wallet", "balance", addrNew.Address.String())
+	assert.Equal(t, "0", balance.String())
 }
 
-func TestAddrLookupAndUpdate(t *testing.T) {
```

Comment:
`address lookup` apparently found the Peer ID of a miner. It's gone now and that functionality exists in the `miner status` command.